spec/controllers/v2/tailorings_controller_spec.rb

```diff
@@ -253,6 +247,42 @@
       end
     end
 
+    context 'with authorization only via cert_auth' do
+      before do
+        allow_any_instance_of(V2::ApplicationController).to receive(:rbac_allowed?).and_call_original
+        allow_any_instance_of(Insights::Api::Common::HostInventory).to receive(:hosts).and_return('results' => [:foo])
+        header = JSON.parse(Base64.decode64(current_user.account.identity_header.raw))
+        header['identity']['auth_type'] = Insights::Api::Common::IdentityHeader::CERT_AUTH
+        request.headers['X-RH-IDENTITY'] = Base64.encode64(header.to_json)
+      end
+
+      let(:parent) do
+        FactoryBot.create(
+          :v2_policy,
+          account: current_user.account,
+          profile: canonical_profile
+        )
+      end
+      let(:extra_params) { { policy_id: parent.id, id: item.id } }
+      let(:canonical_profile) { FactoryBot.create(:v2_profile, :with_rules) }
+      let(:item) do
+        FactoryBot.create(
+          :v2_tailoring,
+          :with_mixed_rules,
+          :with_tailored_values,
+          policy: parent,
+          profile: canonical_profile,
```

Comment:
You definitely don't need to specify both, policy should be always enough as the profile can be looked up automatically if everything is valid.